### PR TITLE
fix: Pin LXD version to LTS

### PR DESCRIPTION
**Pull request by Gu1nness**
## Issue

 * CI is failing due to a bug (https://github.com/canonical/lxd/issues/13774)

## Solution
 * Pin channel to 5.21/stable for now

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -101,6 +101,7 @@ jobs:
       artifact-prefix: packed-charm-cache-true
       cloud: lxd
       juju-agent-version: 3.1.7
+      lxd-snap-channel: 5.21/stable
```
Comment (MiaAltieri) [inline .github/workflows/ci.yaml:106]:
```suggestion
      # TODO: remove when lxd bug is solved:
      # https://github.com/canonical/lxd/issues/13774
      lxd-snap-channel: 5.21/stable
```

**Review by MiaAltieri — DISMISSED** [review]

**Comment by MiaAltieri** [comment]
@Gu1nness - this looks great! Mind bumping the number in the `charm_internal_revision`file?

**Comment by Gu1nness** [comment]
@MiaAltieri Done :)

**Review by MiaAltieri — APPROVED** [review]

**Comment by Gu1nness** [comment]
@MiaAltieri As it has been fixed in `latest/stable` already (see the bug ticket on lxd), I'm closing this.
I have confirmed locally that this is indeed fixed in latest stable.

**Pull request closed without merging**